schemas/expconf/v1/check-epoch-not-used.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "http://json-schema.org/schema#",
```

Comment:
should these all be `http://json-schema.org/draft-07/schema#`?

Reply:
yes, done